### PR TITLE
Deposit dates

### PR DESCRIPTION
**Pull request by Joozty**

### CODE REVIEW

**Comment by viktor-yakubiv** [comment]
The branch seems to be inappropriately rebased.

File: pages/deposit-dates.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
+.table thead > :first-child {
+  border-top-width: 2px;
+}
+
+.container {
+  display: grid;
```
Comment (viktor-yakubiv) [inline pages/deposit-dates.css]: It's better to use flex, in this case and mostly. Check it for mobile.

With flex you can provide grow and shrink properties to make it resize automatically screens size independently. With greed you need a media query.

Add please 18rem minimal width to card component. It should fix the problem here if you use flex.

**Comment by viktor-yakubiv** [comment]
> The branch seems to be inappropriately rebased.

Sorry, this is my fault. I merged #35. You can b are on `master` now.

File: pages/deposit-dates.css (hunk 1 above)
Comment (Joozty) [reply pages/deposit-dates.css]: Fair enough, thanks.

File: components/table/index.jsx [hunk 2]
```diff
@@ -69,7 +77,7 @@ class InfiniteTable extends React.Component {
             value={searchTerm}
           />
         )}
-        <Table>
+        <Table className={className}>
```
Comment (viktor-yakubiv) [inline components/table/index.jsx]: You do not want to pass it using rest props 🙂

File: pages/deposit-dates.css [hunk 3]
```diff
@@ -0,0 +1,49 @@
+/* TODO: Custom media from layout.css doesn't work */
+@custom-media --screen-until-desktop (max-width: 727px);
+
+.table thead > :first-child {
```
Comment (viktor-yakubiv) [inline pages/deposit-dates.css]: Okay, I will note this

File: pages/deposit-dates.css [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  border-top-width: 2px;
+}
+
+.row {
```
Comment (viktor-yakubiv) [inline pages/deposit-dates.css]: I expected to have a single flex container that is actually main. Do you mind if I play with styles a little bit more?

File: components/table/index.jsx (hunk 2 above)
Comment (Joozty) [reply components/table/index.jsx]: true, thanks

File: pages/deposit-dates.css (hunk 4 above)
Comment (Joozty) [reply pages/deposit-dates.css]: Go ahead. You are a CSS Pro. 🙂

**Comment by Joozty** [comment]
@viktor-yakubiv Thank you for review even during the weekend. 🙂

File: pages/deposit-dates.css (hunk 4 above)
Comment (viktor-yakubiv) [reply pages/deposit-dates.css]: Check #41 🙂

**Pull request merged**